<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SequenceWidget</class>
 <widget class="QWidget" name="SequenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>165</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>102</height>
   </size>
  </property>
  <layout class="QGridLayout" name="sequence_grid">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Default values:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="def_values_cmb"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="cyclic_lbl">
     <property name="text">
      <string>Cyclic:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="cyclic_chk">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="start_lbl">
     <property name="text">
      <string>Start:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="start_edt">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhFormattedNumbersOnly</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="increment_lbl">
     <property name="text">
      <string>Increment:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="increment_edt">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhFormattedNumbersOnly</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="minimum_lbl">
     <property name="text">
      <string>Minimum:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="minimum_edt">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhFormattedNumbersOnly</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="maximum_lbl">
     <property name="text">
      <string>Maximum:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="maximum_edt">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhFormattedNumbersOnly</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="cache_lbl">
     <property name="text">
      <string>Cache:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="cache_edt">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhFormattedNumbersOnly</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="owner_col_lbl">
     <property name="text">
      <string>Owner Col.:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
